### PR TITLE
CPFA-1 Create react infrastructure

### PR DESCRIPTION
**Pull request by KenAdams777**
1) set up redux and file structure
2) set up routing

### CODE REVIEW

**Review by MuscularSloth — APPROVED** [review]
Залил себе, проинсталил, запустил. Все работает.
По структуре вроде тоже все нормально.

File: public/index.html [hunk 1]
```diff
@@ -1,43 +1,18 @@
...
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (bejalane) [inline public/index.html:17]: Why do we need to remove this? and also manifest file? We don't need it too much but anyway better to keep it also with manifest.json file

**Review by bejalane — CHANGES_REQUESTED** [review]
Great job, please see some comments.

File: public/index.html (hunk 1 above)
Comment (KenAdams777) [reply public/index.html:17]: Do you mean restore only 'manifest' or icons too?

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
 
 function App() {
-  return (
-    <h1>WELCOME TO THE CODEWAY POETRY APP</h1>
-  );
+	return (
+		<Provider store={store}>
+			<BrowserRouter>
+				<h1>WELCOME TO THE CODEWAY POETRY APP</h1>
```
[the remark this answers is not in the document]
Comment (KenAdams777) [reply src/App.js:12]: Done.

File: src/App.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+h1 {
```
[the remark this answers is not in the document]
Comment (KenAdams777) [reply src/App.scss:1]: Done.

File: src/components/AnnounceAuthor/AnnounceAuthor.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './style__AnnounceAuthor.scss';
```
[the remark this answers is not in the document]
Comment (KenAdams777) [reply src/components/AnnounceAuthor/AnnounceAuthor.js:2]: Done.

File: public/index.html (hunk 1 above)
Comment (bejalane) [reply public/index.html:17]: Up to you, this is not so necessary. You can remove it all or to restore it.

**Review by bejalane — APPROVED** [review]
Great job!

**Pull request merged**